### DISCUSSION TITLE
How to open "new project"

### DISCUSSION THREAD

**Discussion by Miss-Alya** [apache/netbeans#5114] [Q&A]
Hello, I'm a beginner.

I'm using Mac.
I download "Apache Netbeans 16"

everything went great until I wanted to open a "new project"

So, I clicked "open project", and it opened a box, and from categories, I choosed "java with maven" and from Projects, I clicked on "Java Application" And then "Next".

It took a long time until the box showed me this in the picture.

How to fix it?

![‏لقطة الشاشة ١٤٤٤-٠٥-٢١ في ١ ٥٦ ٢٤ ص](https://user-images.githubusercontent.com/120607210/207735806-0c905ef3-efeb-4193-95d5-67f9bcb8f28c.png)

**Comment by mbien** [comment]
that is an interesting place to encounter an error message. Haven't seen that one before.

please attach the log file which you can find in:
`$NETBEANS_USER_DIR/var/log/messages.log`
(you might have to rename it to txt to make github happy)

i don't know where the default dir is on mac os unfortunately, `Help -> About` will list it though.

**Comment by Chris2011** [comment]
You wanted to create a new project, right? Not open a new project, which doesn't work. You either can open a project (no wizard, because you already have one) or you want to create a new project (with wizard). Just to make the steps clear.

**Comment by lkishalmi** [comment]
I guess, that you are running NetBeans on a JRE not a JDK, so the Java development support parts are missing. I do not know if that's possible on Mac or not...
